### DISCUSSION TITLE
ESP32 as node for tasmesh?

### DISCUSSION THREAD

**Discussion by ferbulous** [arendst/Tasmota#21735] [Ideas]
Hi,  is it possible for ESP32 to act as node?
Some of the newer devices are getting ESP32 (or chip swapped with esp32) so I can only have 1 esp32 in a mesh setup and i’d have to replace the extra ones with esp8266

**Comment by sfromis** [comment]
Are you saying that TasMesh does not work for devices using ESP32, as in having done a build without success? The page describing the feature says that the nodes are *typically* ESP8266, but does not suggest *only*.
https://github.com/arendst/Tasmota/blob/development/info/xdrv_57_tasmesh.md

**Reply by ferbulous** [reply]
Hi, meshnode is only for esp8266, if I try the same command on esp32 I just get error.
